tsdb/index/postings_test.go

```diff
@@ -392,8 +392,8 @@ func BenchmarkMerge(t *testing.B) {
 		refs = append(refs, temp)
 	}
 
-	its := make([]Postings, len(refs))
-	for _, nSeries := range []int{1, 10, 100, 1000, 10000, 100000} {
+	its := make([]*ListPostings, len(refs))
+	for _, nSeries := range []int{1, 10, 10000, 100000} {
```

Comment:
Could you explain the reason for testing fewer `nSeries` here?

Reply:
The intermediate ones don't tell us much, just slow down the process.